exopy/tasks/tasks/logic/loop_task.py

```diff
@@ -30,8 +32,8 @@ class LoopTask(InterfaceableTaskMixin, ComplexTask):
     #: is simply a convenience and can be set to None.
     task = Typed(SimpleTask).tag(child=50)
 
-    database_entries = set_default({'point_number': 11, 'index': 1,
-                                    'value': 0})
+    database_entries = set_default({'point_number': 11, 'index': 1, 'value': 0,
+                                    'loop_values':numpy.array([0])})
```

Comment:
It would be nice for the default values to match the other defaults.
```suggestion
                                    'loop_values':np.linspace(0, 1, 11)})
```